asset/src/elasticsearch_reader/elasticsearch_date_range/slicer.ts

```diff
@@ -0,0 +1,325 @@
+
+import { cloneDeep, toString } from '@terascope/job-components';
+import moment from 'moment';
+// @ts-ignore
+import parseError from '@terascope/error-parser';
```

Comment:
Use `parseError` from `@terascope/utils`.
```suggestion
import { parseError } from '@terascope/utils';
```